templates/docker-compose/Makefile.dist

```diff
@@ -11,13 +11,11 @@ COMPOSE_FILES=-f docker-compose.yml -f docker-compose.prod.yml
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(SELF_FILENAME) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: install_cli
-install_cli: install_base ## Install the help desk using the CLI installer.
-	$(DOCKER_BIN) exec -it -u www-data $(WEB_SERVICE_NAME) bash -c '/usr/local/bin/php artisan app:install'
-
 .PHONY: install
-install: install_base ## Install the helpdesk using the web installer.
-	@printf "Open your web browser and complete the installation using the above database details.\n"
+install: configure create_secrets create_volumes ## Install the helpdesk.
+	$(DOCKER_COMPOSE_BIN) $(COMPOSE_FILES) up -d gateway supportpal db redis
+	$(DOCKER_BIN) exec $(DATABASE_SERVICE_NAME) bash -c "while ! (mysqladmin ping -uroot -p$$(cat secrets/db_root_password.txt) --silent); do sleep 1; done"
+	$(DOCKER_BIN) exec -it -u www-data $(WEB_SERVICE_NAME) bash -c "/usr/local/bin/php artisan app:install --db-host=db --db-user=$$(cat secrets/db_user.txt) --db-pass=$$(cat secrets/db_password.txt) --db-name=supportpal"
```

Comment:
Should we replace: `secrets/db_user.txt` with  $(SECRETS_DIR)/db_user.txt

Reply:
Looks good :+1: